core/Cargo.toml

```diff
@@ -22,6 +23,14 @@ multibase = { version = "0.9.1", default-features = false, optional = true }
 serde = { version = "1.0.132", default-features = false, features = ["alloc"], optional = true }
 thiserror = {version = "1.0.25", optional = true }
 
+[target.'cfg(not(target_arch="thumbv6m-none-eabi"))'.dependencies.getrandom]
+version = "0.2.8"
+features = ["custom"] 
+
+[target.'cfg(not(target_arch="thumbv6m-none-eabi"))'.dependencies.quickcheck]
+version = "1"
+optional = true
+
```

Comment:
All this should be be replaces with just

    quickcheck = { version = "1.0", optional = true }

Reply:
Fixed